api/managementpb/dbaas/json/dbaas.json

```diff
@@ -3326,6 +3414,37 @@
                             "format": "int32",
                             "x-order": 0
                           },
+                          "haproxy": {
+                            "description": "HAProxy container parameters.\nNOTE: HAProxy does not need disk size as ProxySQL because the container does not require it.",
```

Comment:
```suggestion
                            "description": "HAProxy container parameters.\nNOTE: HAProxy does not need disk size as ProxySQL does because the container does not require it.",
```

Reply:
@atymchuk I added the missing `does` in .proto file and regenerated the json swagger files.